jobs/build/update-base-images/build.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
```

Comment:
sadly at the moment you need:
```
export REQUESTS_CA_BUNDLE=/etc/pki/tls/certs/ca-bundle.crt
```
otherwise you get this ridiculous error because certifi ignores your default CAs:
`SSLError(SSLError(1, u'[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed`